lisa/util/__init__.py

```diff
@@ -47,7 +48,7 @@
 # source -
 # https://github.com/django/django/blob/stable/1.3.x/django/core/validators.py#L45
 __url_pattern = re.compile(
-    r"^(?:http|ftp)s?://"  # http:// or https://
+    r"^(?:http|s?ftp)s?://"  # http:// or https://
```

Comment:
1. Update the comments to make it consistent with the patterns.
2. Expand the patterns make it easy to read, and avoid invalid pattern like `sftps`, `ftps`. `(?:http|https|sftp|ftp)`

Reply:
Either remove the comments or make it consistent with patterns.